app/components/exp-lookit-stop-recording/component.js

```diff
@@ -154,7 +159,7 @@ export default ExpFrameBaseComponent.extend(ExpandAssets, {
                 _this.send('setTimeEvent', 'warningUploadTimeoutError');
                 _this.send('next');
             }
-        }, 5000));
+        }, 600000));
```

Comment:
Did we intend to leave this timeout this high?

Reply:
Another great catch, thanks for flagging this. I've reduced this from 10 to 5 minutes (and updated the comment about it). I'm not sure what a reasonable duration is for completion of the first part of a multi-part upload on slow connections. I'd like to err on the side of not losing data, and decrease this duration if we get complaints about it. I will also make a note to test this myself on a throttled connection.